private-constructor-checker/src/test/java/com/pushtorefresh/private_constructor_checker/PrivateConstructionCheckerTest.java

```diff
@@ -38,13 +37,11 @@ public void shouldThrowExceptionIfClassDoesNotHaveDeclaredConstructors() {
                     .check();
```

Comment:
Maybe it should be better to rename this test to `shouldThrowExceptionIfClassHasNonDefaultConstructors`?